README_es-AR.md

```diff
@@ -0,0 +1,440 @@
+```
+
+```
+# Bad
+If applied, this commit will used InventoryBackendPool to retrieve inventory backend
+```
+
+### Primer letra en mayúscula
+
+```
+# Good
+Add `use` method to Credit model
+```
+
+```
+# Bad
+add `use` method to Credit model
+```
+
+El motivo detrás de esto es para seguir la regla gramatical que las oraciones deben comenzar en mayúsculas.
+
+El uso de esta práctica puede variar de persona a persona, de equipo a equipo, o incluso de idioma a idioma.
+Mayúsculas o no, es importante adherir a un solo estándar y respetarlo.
+
+### Comunica qué hace el cambio sin necesidad de mirar al código fuente
+
+```
+# Good
+Add `use` method to Credit model
+
+```
+
+```
+# Bad
+Add `use` method
+```
+
+```
+# Good
+Increase left padding between textbox and layout frame
+```
+
+```
+# Bad
+Adjust css
+```
+
+Es muy útil para ayudar a los revisores a entender el razonamiento del autor de los cambios (por ejemplo, cuando hay muchos commits, muchos cambios y refactors)
+
+### Usa el cuerpo del mensaje para explicar "por qué", "para qué", "cómo" y detalles adicionales
+
+```
+# Good
+Fix method name of InventoryBackend child classes
+
+Classes derived from InventoryBackend were not
+respecting the base class interface.
+
+It worked because cart was calling the backend implementation
+incorrectly.
+```
+
+```
+# Good
+Serialize and deserialize credits to json in Cart
+
+Convert the Credit instances to dict for two main reasons:
+
+  - Pickle relies on file path for classes and we do not want to break up
+    everything if a refactor is needed
+  - Dict and built-in types are picklable by default
+```
+
+```
+# Good
+Add `use` method to Credit
+
+Change from namedtuple to class because we need to
+setup a new attribute (in_use_amount) with a new value
+```
+
+El asunto y el cuerpo del mensaje se separan por una línea en blanco.
+Las líneas en blanco adicionales pasan a ser consideradas parte del cuerpo del mensaje.
+
+Caracteres como `-`, `*` y \` pueden aportar legibilidad.
+
+### Evita mensajes genéricos o sin contexto
+
+```
+# Bad
+Fix this
+
+Fix stuff
+
+It should work now
+
+Change stuff
+
+Adjust css
+```
+
+### Limita el número de columnas
+
+[Se recomienda](https://git-scm.com/book/es/v2/Git-en-entornos-distribuidos-Contribuyendo-a-un-Proyecto#r_commit_guidelines) usar un máximo de 50 caracteres para el asunto y 72 para el cuerpo.
+
+### Mantén idioma consistente
+
+Para dueños de proyectos: elijan un idioma y escriban todos los mensajes de commit usando ese idioma. Idealmente debería coincidir con los comentarios del código, la localización por defecto (para proyectos que apliquen), etc.
+
+Para contribuidores: escriban sus mensajes de commits usando el mismo idioma que los que ya existen en el historial de commits.
+
+```
+# Good
+ababab Add `use` method to Credit model
+efefef Use InventoryBackendPool to retrieve inventory backend
+bebebe Fix method name of InventoryBackend child classes
+```
+
+```
+# Good (Portuguese example)
```

Comment:
@diega would you mind to translate these examples to Spanish as well? I think it will be more inclusive.

I've added some Github suggestions using a simple translation using Google Translate. Feel free to use them or provide any example that you find working better.

```suggestion
# Good (Spanish example)
```